puma.sh

```diff
@@ -1,89 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Hence capistrano. Use `cap production puma:restart` from your local machine while in the coursemology dir. There's no need to ssh in to restart it.